sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -2657,6 +2657,29 @@ private <T> Flux<FeedResponse<Document>> createPointReadOperations(
                         BridgeInternal.getClientSideRequestStatics(cosmosItemResponse.getDiagnostics())));
 
                 return Mono.just(feedResponse);
+            })
+            .onErrorResume(throwable -> {
+                if (throwable instanceof CosmosException) {
```

Comment:
should unwrap this throwable, like we do at other places. Please see code for examples on unwrap API from reactor.

Reply:
Resolved in next iteration.